resources/js/components/random/generator/user/AvatarCard.vue

```diff
@@ -5,10 +5,9 @@
     <img v-if="props.src" :src="props.src" class="h-fit" />
     <div
       v-else
-      class="flex h-[30rem] transform items-center justify-center text-center font-['EIGHTY_MILES'] text-8xl text-white"
+      class="flex h-[30rem] transform items-center justify-center text-center font-eighty-miles text-8xl text-white"
     >
-      {{ props.firstname }} <br />
-      {{ props.lastname }}
+      <span>{{ props.firstname }} {{ props.lastname }}</span>
```

Comment:
I meant first and last name each in their own span

Reply:
That doesn't work unfortunately
This approach works. Is it too long it wraps around, else it stays in one line

Reply:
but this approach is not good. And look, the other version will work 
![image](https://github.com/fsr5-fhaachen/portals/assets/16943133/ebe8efc5-3ec7-475e-9439-d143a7596455)

I will do this than in my branch
![image](https://github.com/fsr5-fhaachen/portals/assets/16943133/cd247192-1278-440e-85fd-5eccc2b6fe93)

Reply:
Hö lol I did that and it didnt look like that
But then I will put it in

Reply:
Maybe, just maybe, I was too stupid to save

Reply:
Have you added the `flex-col` class?